src/renderer/components/message/MessageListAndComposer.tsx

```diff
@@ -188,6 +190,67 @@ export default function MessageListAndComposer({ accountId, chat }: Props) {
     [hasOpenDialogs]
   )
 
+  const { jumpToMessage } = useMessage()
+  // TODO add this to the Keyboard shortcuts help (Ctrl + /),
+  // and KEYBINDINGS.md
+  const onCtrlArrow = useCallback(
+    async (e: KeyboardEvent) => {
+      if (!e.ctrlKey) {
+        return
+      }
+      if (e.key !== 'ArrowUp' && e.key !== 'ArrowDown') {
+        return
+      }
+      const quoteMessage = (messageId: number) => {
+        setQuoteInDraft(messageId)
```

Comment:
maybe you could move that logic to the composer? then there would be no need for `__getQuoteInDraft`

Reply:
I thought it's right to place it here based on the fact that in Telegram you can execute this shortcut even if the composer is not focused.
Plus we need access to the list of messages which composer doesn't really have (of course this can be changed but I think it's cleaner this way despite the fact that we mutate the window object)

Reply:
If this doesn't change your mind, I'll move it to the composer.

Reply:
so the shortcut is global? Then it needs to go to keybindings.ts and become a `KeybindAction`

Reply:
and the composer would get the handler for it with `useKeyBindingAction`.

handler function could be cached in an `useCallback`-hook for more efficiency I guess.

Reply:
Well, I mean it doesn't have to be global. It is only active when a chat is active and perhaps when the chat is focused.

Reply:
Currently it is only active when that `window.` function exists.

Reply:
in those cases there is just no handler defined. I'd say.

The keybings system will need a rework in the future, to make a system that we can use to bring keybindings to everything in the app, maybe something stack based, where the top most dialog/context menu has the keyboard shortcut focus.